proto/ethermint/evm/v1/evm.proto

```diff
@@ -216,17 +201,17 @@ message TraceConfig {
   // number of blocks the tracer is willing to go back
   uint64 reexec = 3;
   // disable stack capture
-  bool disable_stack = 5 [ (gogoproto.jsontag) = "disableStack" ];
+  bool disable_stack = 5 [(gogoproto.jsontag) = "disableStack"];
   // disable storage capture
-  bool disable_storage = 6 [ (gogoproto.jsontag) = "disableStorage" ];
+  bool disable_storage = 6 [(gogoproto.jsontag) = "disableStorage"];
   // print output during capture end
   bool debug = 8;
```

Comment:
Are the missing `7`, `4` an issue?

Reply:
No! They are reserved as `DisableMemory`  and `DisableReturnData` are depecrated

Reply:
More info here https://github.com/crypto-facs/ethermint/blob/aeedef9b4bcf42b26a5d1715858f707051413a71/proto/ethermint/evm/v1/evm.proto#L206-L209